### ISSUE TITLE
Documentation links are broken or unhelpful

### ISSUE THREAD

**Issue by codypatnaude** [microsoftgraph/msgraph-sdk-php#139]
The documentation link in the README brings you to a github page containing the HTML source of, what I suppose, is a documentation page. If you go to the wiki and click "Documentation" it leads you here, https://github.com/microsoftgraph/msgraph-sdk-php/tree/master/docs. Which isn't all that useful.

Is there an actual site with documentation that I'm just missing? Could the README be updated to point to that site?

**Comment by MIchaelMainer** [comment]
I agree, that link isn't useful. We don't have a site with documentation other than the [wiki](https://github.com/microsoftgraph/msgraph-sdk-php/wiki) for documentation. I admit this is woefully deficient. 

Please keep voting and commenting your expectations of this documentation.

**Comment by diegotraid** [comment]
Expectations: to be updated and readable?

**Comment by cwolkowski** [comment]
Yes please update and explain how to use all the functions within the SDK... it's taking me hours of research to try to figure out something that should be very simple.

**Comment by PierrickMartos** [comment]
Hi @MIchaelMainer Could you just setup github pages to have a living documentation?

**Comment by Spraynard** [comment]
I think it is really weird that the documentation has been like this for almost two years.

**Comment by PierrickMartos** [comment]
Ping @MIchaelMainer

**Comment by Sebi2020** [comment]
Looks like Microsoft is not interested in supporting PHP. Really unusable without any documentation. Even the official ms graph documentation does not contain any php instructions.
